### PR TITLE
Fine-tune the slideIn animation

### PR DESCRIPTION
**Pull request by kjellr**
### PR Summary: 

Minor adjustments to the slideIn scroll animation. Intent is to strike a natural-feeling balance between subtle and apparent. 

<details><summary>Video Comparison</summary>

#### Before:

https://user-images.githubusercontent.com/1202812/230602988-77319ac0-2668-4382-9343-3e66341a0b94.mp4

#### After:

https://user-images.githubusercontent.com/1202812/230602977-93eb1ed3-8be2-4eb5-99ba-37652c863e3c.mp4

</details>

### What approach did you take?

- Decreasing the `translateY` distance, in an effort to tone down the overall motion.  
- Decreasing the speed of the animation from `--duration-extra-long` to a new `--duration-medium` value. (The standalone `animate--fade-in` speed remains the same, since that animation tends to cover larger areas).

### Testing steps/scenarios

- Visit the demo store, scroll around. 
- Compare to [the existing animation here](https://os2-demo.myshopify.com/?preview_theme_id=139658493974). 

### Demo links

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=139665866774)
- [Editor](https://os2-demo.myshopify.com/admin/themes/139665866774/editor)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

File: assets/base.css [hunk 1]
```diff
@@ -3302,7 +3302,7 @@ details-disclosure > details {
 
   @keyframes slideIn {
     from {
-      transform: translateY(10vh);
+      transform: translateY(3rem);
```
Comment (ludoboludo) [inline assets/base.css:3305]: It's much better with this 👍 

I wonder if we should also reduce the speed at which the animation run 🤔 I think first visits are fine but then when you keep navigating, you don't want to have the user to start wondering weather it's the time it takes for things to load or if we're slowing down their experience for the sake of animation. 

What do you think ? I find `--duration-default` to be pretty decent.

Comment (kjellr) [reply assets/base.css]: `--duration-default` felt a little too fast to me. I pushed a compromise in 128ad25 — let me know how that feels to you.

Comment (ludoboludo) [reply assets/base.css]: Yeah it's looking nice 👌 Should be a good starting point and we can tune it more in the future if needed 👍

Comment (kjellr) [reply assets/base.css]: Great! I've taken this out of draft and opened it up for reviews. 🌟

**Review by ludoboludo — APPROVED** [review]
Looking good, just have one comment

File: assets/base.css [hunk 2]
```diff
@@ -744,9 +744,10 @@ details > * {
 :root {
   --duration-short: 100ms;
   --duration-default: 200ms;
+  --duration-medium: 400ms;
   --duration-long: 500ms;
   --duration-extra-long: 600ms;
```
[the remark this answers is not in the document]
Comment (kjellr) [reply assets/base.css:749]: The long duration is still used for product card hovers (we can probably re-evaluate that later... I suspect that this new 400ms one will also work there). The extra-long duration is also still used when we fade-in large items on scroll, so they don't appear to flash.

**Review by metamoni — APPROVED** [review]

File: assets/base.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (stufreen) [reply assets/base.css:749]: I think `extra-long` was only used for the new animations, right? Can it be removed?

**Review by stufreen — APPROVED** [review]
LGTM. I think it's a bit snappier.

One thing you could experiment with is different easings, but longer durations... I find cubic bezier better for things like this because it takes a bit more time settling into place. E.g. https://cubic-bezier.com/#.17,.67,.55,.98

File: assets/base.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (kjellr) [reply assets/base.css:749]: We're still using `extra-long` for the standalone [opacity fade-in](https://github.com/Shopify/dawn/blob/f9bb7bcf390989ec2f04de6ee1c8430c06d0c0f3/assets/base.css#L3284). We use this for large images, and I found it starts to seem like flashing when we move to anything quicker.

**Comment by kjellr** [comment]
While the improvement here felt pretty good, I thought I'd try another pass using @stufreen's `cubic-bezier()` suggestion. In [aba46c1](https://github.com/Shopify/dawn/pull/2499/commits/aba46c11d67d0a26a2a558e689b76875827a4dab), I used a slightly longer transition, with a tweaked version of `ease-out`. I also slightly shortened the timing for items that cascade in. 

This version feels pretty good to me too. If you're scrolling slow, the animation is there but not overwhelming. If you're scrolling quickly, the animation's pretty subtle, and I kinda like that. 

Give it a try and let me know what you think! It might also be worth testing this once https://github.com/Shopify/dawn/pull/2405 and https://github.com/Shopify/dawn/pull/2410 are merged in so we have a better sense of it all.

**Review by stufreen — APPROVED** [review]
Looks great! Subtly better now

**Review by ludoboludo — APPROVED** [review]

**Pull request merged**